manim/utils/tex_file_writing.py

```diff
@@ -21,11 +21,14 @@ def tex_hash(expression):
     return hasher.hexdigest()[:16]
 
 
-def tex_to_svg_file(expression, source_type):
-    tex_template = config["tex_template"]
+def tex_to_svg_file(expression, source_type, tex_template=None):
```

Comment:
```suggestion
def tex_to_svg_file(expression:str , source_type, tex_template:str =None):
```
What should be acceptable value of `source_type` ?